ui/settingsdlg.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>836</width>
-    <height>665</height>
+    <width>873</width>
+    <height>840</height>
```

Comment:
I think it's simply this change. Revert to the old value (or slightly increase if needed in advanced tab).

Reply:
Ah, that would make sense. I bet what happened is me opening the file in Qt Designer on my Kubuntu machine caused everything to get taller (and probably wider too) because of the size of Breeze widgets vs. the size of widgets when using Qt on XFCE. If I look at it in Designer with Breeze, it looks perfectly fine, but all the buttons are a different size here too.

I tried forcing the use of Fusion and it seems to have made things a bit smaller... but not by much. I guess I need to find out what theme is being used in Qubes OS itself and use that (or just develop on my Qubes OS machine rather than my main development rig).

Reply:
Maybe simply try to manually edit those two to the old values?

Reply:
yep, I will, just trying to figure out how to avoid this happening in the future.

Reply:
Tested, that fixed it.